src/components/views/MovieCard.vue

```diff
@@ -0,0 +1,78 @@
+<script>
+import { defineComponent } from 'vue';
+import TheTag from '../common/TheTag.vue';
+import SectionTitle from '../common/SectionTitle.vue';
+    
+export default defineComponent({
+    components: { TheTag, SectionTitle },
+    props: {
+        movie: Object,
```

Comment:
It's a good practice to provide as much information as we can about the props so that we would get the help we need if we make a mistake.
In this case, this component doesn't make sense to not have any movie information so we could mark this prop as `required: true` 

```suggestion
        movie: {
          type: Object,
          required: true
        },
```

Reply:
Got it! 🙂 👍